src-v2/Judge/NTS.Judge.Blazor/Pages/Setup/Competitions/CompetitionForm.razor

```diff
@@ -45,7 +42,7 @@
     {
         RegisterInjector(nameof(_model.Name), () => _nameField);
         RegisterInjector(nameof(_model.StartTime), () => _timeField);
-        RegisterInjector(nameof(_model.Type), () => _typeField);
+        //RegisterInjector(nameof(_model.Type), () => _typeField);
```

Comment:
Is there a problem with the validation?

Reply:
Currently, the RegisterInjector only accepts `Func<MudBaseInput<TInput>>` or `Func<MudPicker<TInput>>` as a second argument. We need to extend it to also support Custom components because the _typeField is no longer of type `MudSelect<CompetitionType>` but `NotSelect<CompetitionType>` instead.

I have left a comment:
// TODO: Implement Validation injection for custom input components
at line 17 NotSelect.razor to ensure the needed improvements aren't forgotten.

Reply:
Makes sense. It's probably going to stay like that for the demo version and then for NTS 1 we'll probably end up reworking the validation anyway.